tests/datatypes/test_bytes.py

```diff
@@ -61,6 +61,42 @@ def test_find(self):
             print(b'abcd'.find(b'ab', 3, 4))
         """)
 
+    def test_isdigit(self):
+        self.assertCodeExecution("""
+            for byt in [b'123', b'678', b'99999', b'abc', b'2s3x', 'a2b3', b'2S3X', 'A2b3', b'6', b'k', 'GHI', 'Jkl']:
+                print(byt.isdigit())
+        """)
+
+    def test_islower(self):
+        self.assertCodeExecution("""
+            for byt in [b'123', b'678', b'99999', b'abc', b'2s3x', 'a2b3', b'2S3X', 'A2b3', b'6', b'k', 'GHI', 'Jkl']:
```

Comment:
Hey @mistermocha , thanks for the PR!
This looks good, just one small thing here: could you add a test with punctuation as well?

This works in CPython:

```
>>> b'a..'.islower()
True
```

The current implementation is returning `False`.

`islower` should return True if there is at least one character that is letter and lower case, and all of the others are not upper case.

The same applies to `isupper`, with the cases swapped.